app/javascript/packs/components/RangeOption.jsx

```diff
@@ -1,20 +1,36 @@
 import React from 'react';
+import { debounce } from 'lodash';
 
 class RangeOption extends React.Component {
-  checkRange(num) {
+  componentDidMount() {
+    this.trySaveAnswer = debounce(function (id, num) {
+      this.props.onSave(id, num);
+    }, 3000);
+
+    if (this.props.answer) {
+      this.onChange(this.props.answer.text);
+    }
+  }
+
+  updateAnswer(id, num, force) {
```

Comment:
A note ahead for this function when we use flow; you may want to default this value to `false` so that is not `undefined` (a non-boolean) when you specify its type!

Defaulting in ES6 looks like `(force=false)`